ProcessLib/ThermoMechanics/ThermoMechanicsFEM.h

```diff
@@ -246,11 +253,12 @@ class ThermoMechanicsLocalAssembler
                     DisplacementDim>::value>;
 
             // assume isotropic thermal expansion
+            const double T_ip = N.dot(T);  // T at integration point
```

Comment:
Very minor note: `N * T` would work, because it's row times column vector.

Reply:
Not sure, if a 1x1 matrix object is convertible to double....

Reply:
Then I keep it.

Reply:
https://godbolt.org/g/QhizDv

1x1 matrix is not convertible to double.